m4_sessions/automation/session_one/one_elijah_annie_schutte_devin.md

```diff
@@ -1,3 +1,71 @@
+*   ["Jerry Kaplan: Humans Need Not Apply at Google"](https://www.youtube.com/watch?v=JiiP5ROnzw8) (Google: November 12, 2015)
+
+#### Cons
+* ["Six Jobs Are Lost for Every Robot Added to the Workforce"](https://futurism.com/new-study-finds-that-six-jobs-are-lost-for-every-robot-added-to-the-workforce/)
+* [Why Do Americans Work So Much?](https://www.theatlantic.com/business/archive/2016/01/inequality-work-hours/422775/)(The Atlantic: Jan 7, 2016)
+* [Why do we still spend so much time at work?](http://www.ecnmy.org/learn/your-workplace/where-we-work/were-producing-stuff-faster-and-cheaper-than-ever-why-do-we-still-spend-so-much-time-at-work/)
+
+### Main Questions
+
+1. How serious an impact do you think automation will have on our economy from now until you retire? 
+2. Where do you think you fall across the spectrum of people that will be affected by these changes? 
+3. What do you think it means to be a responsible citizen around this issue? 
+4. What role can Turing play in advancing civic responsibility in response to the challenges automation will bring? 
+
+### Groupings
+
+* Two groups, number off 1 and 2.
+
+## Session Outline
+
+| Timing        | Facilitator Talking Points           | Participant/Facilitator Actions  |
+| ------------- | ------------- | ----- |
+
+| 5 min | Introduction: All the feels re: the big A word. | At facilitator's prompting, students should introduce themselves to others in their group. |
+
+| 5 min | Warm Up: Kick off an activity where students will organize in the room along a spectrum of their attitudes toward automation. Describe one end as representing people who feel strongly that automation is, as Elon Musk says, "summoning a demon." On the other end, people who feel automation is going to take humanity to new, positive, exciting potential. | Facilitator will instruct students to line up in between these poles as appropriate. At facilitator's prompting, students should line up along the spectrum of attitudes toward automation. |
+
+| 2 min | Introduce video and reflection questions to students. <br> 1) How serious an impact do you think automation will have on our economy from now until you retire? <br> 2) Where do you think you fall across the spectrum of people that will be affected by these changes? <br> 3) What do you think it means to be a responsible citizen around this issue? <br> 4) What role can Turing play in advancing civic responsibility in response to the challenges automation will bring?  <br> | Facilitor will display questions on screen. Students get out paper and pen to prepare for video. |
+
+| 15 min | Kick off ["Humans Need Not Apply"](https://www.youtube.com/watch?v=7Pq-S557XQU) video. | Students should take notes during the video.  Ask them to put away cell phones/computers if needed. |
```

Comment:
if this is in the pre-reading session, then do you want to show it during session time?